selfdrive/common/util.cc

```diff
@@ -84,3 +54,20 @@ int set_core_affinity(int core) {
   return -1;
 #endif
 }
+
+namespace util {
+
+std::string read_file(const std::string& fn) {
+  std::ifstream ifs(fn, std::ios::binary | std::ios::ate);
+  if (ifs) {
+    std::ifstream::pos_type pos = ifs.tellg();
+    std::string result;
+    result.resize(pos);
+    ifs.seekg(0, std::ios::beg);
+    ifs.read(result.data(), pos);
+    if (ifs) return result;
+  }
+  return {};
```

Comment:
why not? think this is more readable
```suggestion
  return "";
```